packages/ui-extensions/src/surfaces/customer-account/globals.ts

```diff
@@ -1,3 +1,4 @@
+import {FullExtensionNavigation} from './api';
 import type {ExtensionTargets} from './extension-targets';
 
 export interface ShopifyGlobal {
```

Comment:
There's should be no `reload` any more unless you're keeping it around

Reply:
we never had `reload` in the api, right? 🤔